helm/templates/40_stats.yaml

```diff
@@ -1,153 +1,78 @@
 apiVersion: apps/v1
 kind: StatefulSet
 metadata:
-  name: zookeeper
+  name: clickhouse-bulk
   namespace: argoflow
 spec:
   replicas: 1
-  serviceName: zookeeper-service
+  serviceName: clickhouse-bulk-service
   selector:
     matchLabels:
-      component: zookeeper
+      component: clickhouse-bulk
   template:
     metadata:
       labels:
-        component: zookeeper
+        component: clickhouse-bulk
     spec:
       containers:
-      - name: zookeeper
-        image: zookeeper:3.4.9
+      - name: clickhouse-bulk
+        image: nikepan/clickhouse-bulk:1.3.7
+        resources:
+          requests:
+            cpu: 100m
+            memory: 100Mi
```

Comment:
@BigBreadman Let's remove these resources restriction because k8s will kill the pod every time it reaches one of these limits